# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/lib-conf.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-257/src/basic/constants.h.vanilla	2025-01-09 13:54:38.046666642 +0100
+++ systemd-257/src/basic/constants.h	2025-01-09 13:54:48.566666655 +0100
@@ -61,13 +61,13 @@
 #define CONF_PATHS_NULSTR(n)                    \
         "/etc/" n "\0"                          \
         "/run/" n "\0"                          \
-        "/usr/local/lib/" n "\0"                \
+        "/lib/" n "\0"                \
         "/usr/lib/" n "\0"
 
 #define CONF_PATHS(n)                           \
         "/etc/" n,                              \
         "/run/" n,                              \
-        "/usr/local/lib/" n,                    \
+        "/lib/" n,                    \
         "/usr/lib/" n
 
 #define CONF_PATHS_STRV(n)                      \
